src/cli.rs

```diff
@@ -801,7 +802,8 @@ impl clap::Args for Exec {
                 .conflicts_with_all(["exec", "list_details"])
                 .help("Execute a command with all search results at once")
                 .long_help(
-                    "Execute the given command once, with all search results as arguments.\n\
+                    "Execute the given command once, with all search results as arguments, in the order \
```

Comment:
"in the order they were found" is only true if `-j 1` is specified. In general the order is non-deterministic. And even in the `-j 1` case, the actual order is an implementation detail.  I'd rather say something to the effect of "You sholdn't rely on the order the results are given in".